src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -58,7 +59,10 @@ public void setupDriveCommands(OperatorInterface oi,
             PositionMaintainerCommand positionMaintainer,
             PositionDriveWithJoysticksCommand positionDrive,
             VelocityDriveWithJoysticksCommand velocityDrive) {
-        resetHeading.setHeadingToApply(0);
+        resetHeading.setHeadingToApply(
+                drive.getDriverOrientedDriveEnabled()
```

Comment:
⭐  consider factoring some of this out, it's a lot to read in 1 big nested ternary